packages/data-context/src/actions/WizardActions.ts

```diff
@@ -51,9 +52,21 @@ export class WizardActions {
     }
   }
 
-  setBundler (bundler: WizardBundler | null) {
+  getNullableBundler (bundler: 'vite' | 'webpack' | null) {
```

Comment:
So the reason for all the change around `bundler` is previously we did something like:

```js
export const WIZARD_DEPENDENCY_WEBPACK = {
  type: 'webpack',
  name: 'Webpack',
  package: 'webpack',
  installer: 'webpack',
  description: 'Webpack is a module bundler',
  minVersion: '^=4.0.0 || ^=5.0.0',
} 

supportedBundlers: [WIZARD_DEPENDENCY_WEBPACK]
```

But for a third party to do this, we'd need to expose the `WIZARD_DEPENDENCY_WEBPACK`, which is not really ideal or in line with what we currently do and how we define public APIs. In `cypress.config`, we just do

```js
bundler: 'webpack'
```

I reworked the internal API (which is also now the public API) to use a string, `webpack | vite`, instead of the complex `WIZARD_DEPENDENCY_WEBPACK` object. It's way more ergonomic. If we do need the full object, we just use `getNullableBundler` to grab it.